dev/com.ibm.ws.jpa_spec21_fat/fat/src/com/ibm/ws/jpa/TestTXDDSynchronization.java

```diff
@@ -0,0 +1,254 @@
+    }
+
+    private final static String EJB_CONTEXTROOT = "TxSynchronizationEJBDD";
+    private final static String WEB_CONTEXTROOT = "TxSynchronizationDD";
+
+    @Server("JPAServerTxSynchronization")
+    @TestServlets({
+                    @TestServlet(servlet = TxSynchronizationDDWebTestServlet.class, path = WEB_CONTEXTROOT + "/" + "TxSynchronizationDDWebTestServlet"),
+                    @TestServlet(servlet = TxSynchronizationDDCMTSSpecificWebTestServlet.class, path = WEB_CONTEXTROOT + "/" + "TxSynchronizationDDCMTSSpecificWebTestServlet"),
+
+                    @TestServlet(servlet = TxSynchronizationDDEJBSLTestServlet.class, path = EJB_CONTEXTROOT + "/" + "TxSynchronizationDDEJBSLTestServlet"),
+                    @TestServlet(servlet = TxSynchronizationDDCMTSSpecificEJBSLTestServlet.class,
+                                 path = EJB_CONTEXTROOT + "/" + "TxSynchronizationDDCMTSSpecificEJBSLTestServlet"),
+
+                    @TestServlet(servlet = TxSynchronizationDDEJBSFTestServlet.class, path = EJB_CONTEXTROOT + "/" + "TxSynchronizationDDEJBSFTestServlet"),
+                    @TestServlet(servlet = TxSynchronizationDDCMTSSpecificEJBSFTestServlet.class,
+                                 path = EJB_CONTEXTROOT + "/" + "TxSynchronizationDDCMTSSpecificEJBSFTestServlet"),
+
+                    @TestServlet(servlet = TxSynchronizationDDEJBSFExSyncTestServlet.class, path = EJB_CONTEXTROOT + "/" + "TxSynchronizationDDEJBSFExSyncTestServlet"),
+                    @TestServlet(servlet = TxSynchronizationDDEJBSFExUnsyncTestServlet.class, path = EJB_CONTEXTROOT + "/" + "TxSynchronizationDDEJBSFExUnsyncTestServlet"),
+
+                    @TestServlet(servlet = TxSynchronizationDDCMEXSpecificSyncTestServlet.class,
+                                 path = EJB_CONTEXTROOT + "/" + "TxSynchronizationDDCMEXSpecificSyncTestServlet"),
+                    @TestServlet(servlet = TxSynchronizationDDCMEXSpecificUnsyncTestServlet.class,
+                                 path = EJB_CONTEXTROOT + "/" + "TxSynchronizationDDCMEXSpecificUnsyncTestServlet"),
+
+                    @TestServlet(servlet = TxSynchronizationDDCMEXSpecificRunnerTestServlet.class,
+                                 path = EJB_CONTEXTROOT + "/" + "TxSynchronizationDDCMEXSpecificRunnerTestServlet"),
+
+                    @TestServlet(servlet = TxSynchronizationDDCMTCMTSSpecificEJBSLTestServlet.class,
+                                 path = EJB_CONTEXTROOT + "/" + "TxSynchronizationDDCMTCMTSSpecificEJBSLTestServlet"),
+
+                    @TestServlet(servlet = TxSynchronizationDDCMTCMEXSpecificEJBSLTestServlet.class,
+                                 path = EJB_CONTEXTROOT + "/" + "TxSynchronizationDDCMTCMEXSpecificEJBSLTestServlet"),
+
+    })
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        PrivHelper.generateCustomPolicy(server, FATSuite.JAXB_PERMS);
+        bannerStart(TestOLGH8820_Web.class);
```

Comment:
Looks like a copy/paste mistake

Reply:
Noted, will correct in an update commit.